lib/worker.js

```diff
@@ -34,7 +44,7 @@ Communication.on('JOB', function(job) {
   let configFile = null
   global.__LINTER_RESPONSE = []
 
-  configFile = find(params.fileDir, '.eslintrc')
+  configFile = find(params.fileDir, '.eslintrc', 1)
```

Comment:
Are you sure the config file supports multiple files as a parameter? The wording in [the documentation](http://eslint.org/docs/user-guide/command-line-interface) seems to indicate this should be a single file.